File: financial/Models/CategoryDTO.cs
using System.Collections.Generic;

namespace Models
{
    public class CategoryDTO : BaseEntity
    {
        public int EstablishmentId { get; set; }
        public string ImageName { get; set; }
        public virtual IEnumerable<Category> Categorys { get; set; }
        public virtual IEnumerable<OpeningHours> OpeningHours { get; set; }
        public CategoryDTO()
        {
        }
    }
}

File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/CategoryDTO.kt
package com.br.app.deliverydelicatessen.model

import java.util.ArrayList

data class CategoryDTO(
    var establishmentId: Int = 0,
    var imageName: String = "",
    var categorys: MutableList<Category> = ArrayList(),
    var openingHours: MutableList<OpeningHours> = ArrayList()
)